addons/website/static/src/scss/website.ui.scss

```diff
@@ -515,6 +515,27 @@ body .modal {
     }
 }
 
+// FOOTER
+.o_footer_copyright {
+    padding: 8px;
+    .js_language_selector {
+        display: inline-block;
+    }
+    @media (min-width: $screen-sm-min) {
+        .row {
+            @include o-flex-display;
+            > div {
+                margin: auto 0;
+            }
+        }
+    }
+    @media (max-width: $screen-xs-max) {
+        .row > div {
+            text-align: center;
+        }
+    }
+}
```

Comment:
The footer is not part of the website ui... so this is not the right file to put this.

Reply:
Moved it to website.scss